include/aktualizr-lite/aklite_client_ext.h

```diff
@@ -0,0 +1,30 @@
+#ifndef AKTUALIZR_LITE_CLIENT_EXT_H_
+#define AKTUALIZR_LITE_CLIENT_EXT_H_
+
+#include <cstdlib>
+#include <string>
+
+#include "json/json.h"
+
+#include "aktualizr-lite/tuf/tuf.h"
+
+#include "aktualizr-lite/api.h"
+
+/**
+ * @brief contains additional methods that consolidate functionality
+ * making it reusable between the main aklite daemon and other tools.
+ *
+ * Those methods may eventually be part of the supported API but, for now,
+ * they are likely to have they signature and behavior changed.
+ */
+class AkliteClientExt : public AkliteClient {
+  // Inherit constructors
+  using AkliteClient::AkliteClient;
+
+ public:
+  CheckInResult GetTargetToInstall(const LocalUpdateSource* local_update_source, int version,
```

Comment:
maybe it makes sense to make the `local_update_source` as the last parameter with a default  `nullptr` value?

Reply:
All parameters now have default values, and on the typical case, no parameters would be passed when calling the method (the `current` parameter was removed).
I tend to keep the order we have now, leaving `local_update_source` as the first one. In the scenario where the parameters are explicitly set (inside `cli.cc`), `local_update_source` seems to be the most relevant.